std/haxe/macro/CompilationServer.hx

```diff
@@ -86,5 +95,17 @@ class CompilationServer {
 	static public function invalidateFiles(filePaths:Array<String>) {
 		@:privateAccess Compiler.load("server_invalidate_files", 1)(filePaths);
 	}
+
+	/**
+		Get current compilation server stats counters.
+
+		Can be called at different compilation stages, or even in the middle of
+		a macro execution.
+
+		Counters are reset at the beginning of each request.
```

Comment:
Well.. that makes me think this function should probably not live in `CompilationServer.hx`..

Reply:
I know what you mean, but I think it's fine to have it there.

Reply:
Guess we can merge then?